scripts/set-pusher-suffix.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# a script that sets Pusher room suffix (for internal usage)
+
+# config file to be parsed for the suffix (relative to current project root)
+CONFIG_FILE="../Web-Expensify/_config.local.php"
+
+# use the suffix only when the config file can be found
+if [ -f "$CONFIG_FILE" ]; then
+    echo "Using PUSHER_DEV_SUFFIX from $CONFIG_FILE"
+
+    PATTERN="PUSHER_DEV_SUFFIX.*'(.+)'"
+    while read -r line; do
+      if [[ $line =~ $PATTERN ]]; then
+        PUSHER_DEV_SUFFIX=${BASH_REMATCH[1]}
+        echo "Found suffix: $PUSHER_DEV_SUFFIX"
+        echo "Updating .env"
+
+        # delete any old suffix value and append the new one
+        sed -i '' '/^PUSHER_DEV_SUFFIX/d' '.env' || true
+        echo "PUSHER_DEV_SUFFIX=${PUSHER_DEV_SUFFIX}" >> .env
+
+      fi
+    done < "$CONFIG_FILE"
+fi
```

Comment:
The original idea for this script was to use it to `export` an ENV variable `PUSHER_DEV_SUFFIX` and not write it to `.env`, but the variable is not picked up in `react-native` unless it's part of `.env` (even when it's successfully exposed to sister scripts)

This script can be replaced by a nodejs script doing the same thing if you think it'll be cleaner

Reply:
> the variable is not picked up in `react-native` unless it's part of `.env` (even when it's successfully exposed to sister scripts)

That's quite annoying :D But good to know.

> This script can be replaced by a nodejs script doing the same thing if you think it'll be cleaner

I think I'd prefer a node script if it'll do the same thing, mainly because it seems it would be easier to maintain & get help on (if edits are needed in the future)

Reply:
Aaactually since we have so many other shell scripts in `/scripts` I think it's fine to keep it `.sh`